src/CRUD.php

```diff
@@ -152,13 +152,13 @@ public function setModel(Model $m, $fields = null): Model
     protected function initActionExecutor(Generic $action)
     {
         $executor = $this->getExecutor($action);
-        $executor->onHook('afterExecute', function ($ex, $return, $id) use ($action) {
+        $executor->onHook(ActionExecutor\Basic::HOOK_AFTER_EXECUTE, function ($ex, $return, $id) use ($action) {
```

Comment:
Probably better to use `$executor->onHook($executor::HOOK_AFTER_EXECUTE, ...)` here.